src/modules/Auth/services/auth.ts

```diff
@@ -1,26 +1,63 @@
-export interface User {
-  name: string;
-  email: string;
-  password?: string;
+import Keycloak from 'keycloak-js';
+
+const isProduction = process.env.NODE_ENV === 'production';
+const configUrl = isProduction ? 'keycloak.json' : 'keycloak.dev.json';
+const onLoad: Keycloak.KeycloakOnLoad = 'login-required';
+const _kc = Keycloak(configUrl);
+
+function init(appEntrypoint: () => void): void {
+  _kc
+    .init({
+      onLoad,
+    })
+    .then((auth) => {
```

Comment:
just a suggestion, prefer to use async/await, which makes the code easier to read.

Reply:
Cool! Gonna change it